### DISCUSSION TITLE
Unable to access my head node today

### DISCUSSION THREAD

**Discussion by msovara** [chpc-tech-eval/scc#69] [Q&A] [answered]
![MobaXterm screenshot](https://github.com/chpc-tech-eval/chpc24-scc-nmu/assets/18695654/a83d8589-2234-4779-aeba-c552e34c5bf5)

**Comment by mmhashatsi** [comment, marked as answer]
Hey @msovara go to the home directory that is under your name that we used yesterday, and try ssh from there
